resources/js/components/GlobalSearch.vue

```diff
@@ -0,0 +1,282 @@
+<script>
+import ThemeWidget from './Theme/ThemeWidget';
+import PersonWidget from './PersonWidget';
+import MapWidget from './Map/MapWidget';
+import { ContentLoader } from 'vue-content-loader';
+import location from '../api/location';
+import people from '../api/people';
+import topics from '../api/topics';
+import intersectionObserverMixin from '../mixins/intersectionObserverMixin';
+export default {
+    name: 'GlobalSearch',
+    mixins: [
+        intersectionObserverMixin
+    ],
+    components: {
+        ThemeWidget,
+        PersonWidget,
+        MapWidget,
+        ContentLoader
+    },
+    data () {
+        return {
+            onFocus: false,
+            loading: false,
+            observableBlock: '.ris-load-element',
+            inputPlaceholder: 'Suche nach Themen, Vorlagen, Sitzungen...',
+            optionsList: [
+                'Bewohnerparken',
+                'Radwege',
+                'Brückensanierung',
+                'Umweltzone',
+                'Öffentliche Parkplätze',
+                'Schulen',
+                'Klimanotstand',
+                'eRoller'
+            ],
+            inputValue: '',
+            activeTab: 'Alle',
+            topicsList: [],
+            topicsCount: 0,
+            paginationTopics: 0,
+            topicsFlag: true,
+            peopleList: [],
+            peopleCount: 0,
+            paginationPeople: 0,
+            peopleFlag: true,
+            locationList: [],
+            locationCount: 0,
+            paginationLocation: 0,
+            locationFlag: true,
+            debounce: null
+        };
+    },
+    computed: {
+        navigationList () {
+            const list = [];
+            if (this.topicsList.length) list.push('Themen');
+            if (this.locationList.length) list.push('Orte');
+            if (this.peopleList.length) list.push('Personen');
+            return ['Alle', ...list];
+        },
+        isNothing () {
```

Comment:
i would name it something like `isEmptySearchResults` or similar, when i first saw `isNothing` it really meant nothing to me

Reply:
is renamed